src/scripts/components/security-rules.js

```diff
@@ -23,7 +23,8 @@ export function initializeSecurityRules() {
         let results = [];
 
         if (filterCategoryValue && filterCategoryValue !== 'all') {
-            results = document.querySelectorAll(`.${filterCategoryValue}`);
+            const val = (filterCategoryValue.startsWith('.')) ? filterCategoryValue.slice(1) : filterCategoryValue;
+            results = document.querySelectorAll(decodeURI(`.${val}`));
```

Comment:
Note: fixing for error `Failed to execute 'querySelectorAll' on 'Document': '..cat-aws' is not a valid selector.`